public/env-config.js

```diff
@@ -0,0 +1,13 @@
+window._env_ = {
+  NODE_ENV: 'test',
+  PUBLIC_URL: undefined,
+  REACT_APP_PROFILE_GRAPHQL: 'https://helsinkiprofile.test.kuva.hel.ninja/graphql/',
+  REACT_APP_HELSINKI_ACCOUNT_AMR: 'helusername',
+  REACT_APP_APPLICATION_NAME: 'open-city-profile-ui',
+  REACT_APP_OIDC_AUTHORITY: 'https://api.hel.fi/sso/openid/',
+  REACT_APP_OIDC_CLIENT_ID: 'https://api.hel.fi/auth/helsinkiprofile-ui',
+  REACT_APP_PROFILE_AUDIENCE: 'https://api.hel.fi/auth/helsinkiprofile',
+  REACT_APP_OIDC_SCOPE: 'openid profile https://api.hel.fi/auth/helsinkiprofile',
+  REACT_APP_SENTRY_DSN: 'https://8b5b23e2171b42cd8617e2b1ad7353b6@sentry.hel.ninja/63',
+  REACT_APP_VERSION: '1.1.0'
+}
```

Comment:
Missing newline

Reply:
Adding also .editorconfig that handles these.